low-level/framework/base/sspl_constants.py

```diff
@@ -49,10 +52,39 @@
 ENCL_DOWNLOAD_LOG_MAX_RETRY = 60
 ENCL_DOWNLOAD_LOG_WAIT_BEFORE_RETRY = 15
 
-node_key_id = node_id
-CONSUL_HOST = consulhost
-CONSUL_PORT = consulport
+# required only for init
+component = 'sspl/config'
+file_store_config_path = '/etc/sspl.conf'
+salt_provisioner_pillar_sls = 'sspl'
+salt_uniq_attr_per_node = ['cluster_id']
+salt_uniq_passwd_per_node = ['RABBITMQINGRESSPROCESSOR', 'RABBITMQEGRESSPROCESSOR', 'LOGGINGPROCESSOR']
+
+# Initialize to default values
+node_key_id = 'srvnode-1'
+CONSUL_HOST = '127.0.0.1'
+CONSUL_PORT = '8500'
 CONSUL_ERR_STRING = '500 No cluster leader'
+
+# Check if SSPL is configured
+if os.path.exists(SSPL_CONFIGURED):
```

Comment:
ideally we dont to maintain too much logic in this constants file, as its meant for constants specifically. See if we can maintain in different file & then just import final values. 
Not a must right now, keep a TODO if needed

Reply:
@cdeshmukh `its meant for constants specifically` shouldn't those values which are not constant(values which are coming from some file based on logic) come from config ?